### PR TITLE
Added Home inSignUp

### PR DESCRIPTION
**Pull request by opticod**
#411 Home Added

### CODE REVIEW

File: app/src/main/java/com/womenwhocode/womenwhocode/activities/SignUpEmailActivity.java [hunk 1]
```diff
@@ -41,6 +41,14 @@ private static boolean isValidPassword(CharSequence target) {
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_email_sign_up);
+        findViewById(R.id.home).setOnClickListener(new View.OnClickListener() {
```
Comment (zassmin) [inline app/src/main/java/com/womenwhocode/womenwhocode/activities/SignUpEmailActivity.java]: please factor out into a wrapper method within the activity

File: app/src/main/res/layout/activity_email_sign_up.xml [hunk 2]
```diff
@@ -6,7 +6,14 @@
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
                 android:padding="@dimen/activity_horizontal_padding">
-
+        <ImageView
```
Comment (zassmin) [inline app/src/main/res/layout/activity_email_sign_up.xml]: nitpick: please match indents

**Comment by zassmin** [comment]
@opticod thanks for the PR! It's looking good and :+1: on the solution!

Few things: 
* [ ] Let's use this home icon - https://design.google.com/icons/#ic_home
* color [darkGray](https://github.com/WomenWhoCode/WomenWhoCode/blob/master/app/src/main/res/values/colors.xml#L7)

and a few more of the comments I made directly on the code.

**Comment by opticod** [comment]
@zassmin done.

**Comment by zassmin** [comment]
@opticod looking good! Please set icon size to 24dp for a material look, https://www.google.com/design/spec/style/icons.html#icons-system-icons. I would create a var in dimens file `icon_size`. 

Also, we use 4 spaces on xml files not 8.

File: app/src/main/res/layout/activity_email_sign_up.xml [hunk 3]
```diff
@@ -7,6 +7,15 @@
                 android:layout_height="match_parent"
                 android:padding="@dimen/activity_horizontal_padding">
 
+    <ImageView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:src="@drawable/home"
```
Comment (zassmin) [inline app/src/main/res/layout/activity_email_sign_up.xml]: since it's an icon, let's rename to `ic_home` for consistency.

**Comment by opticod** [comment]
Ok I will reformat

**Comment by opticod** [comment]
Ok,will correct the spaces.

**Comment by opticod** [comment]
@zassmin , the layout is lagging materialistic design...like missing a toolbar,color,sizes , etc. Lets come up with a nice materialistic design. I will be helping. Can you please provide me the : 

- Primary color
- Secondary color
- Accent color for this app.

**Comment by zassmin** [comment]
@opticod the line spacing only needed to be fixed for the code related to this PR. Please fix. 

Also, adjusting the coloring and sizing for the rest of this app is outside of the scope for this PR.

**Comment by zassmin** [comment]
I do have a separate issue for line spacing in general: https://github.com/WomenWhoCode/WomenWhoCode/issues/415

**Comment by opticod** [comment]
Ok refactoring I have done it in #416 and fixed this commit also. @zassmin

**Comment by zassmin** [comment]
awesome @opticod! :+1: :8ball: :tada:

**Pull request merged**